src/languages/julia.js

```diff
@@ -84,7 +84,7 @@ function(hljs) {
 
   // placeholder for recursive self-reference
   var DEFAULT = {
-    aliases: ['jldoctest'],
+    aliases: ['julia.console'],
```

Comment:
I didn't mean it's a programmed feature, just a convention :smile: Examples that I can find:

* [pf.js](https://github.com/isagalaev/highlight.js/blob/905119aad47d4bb3d4fbaa14df7598034dccb6a3/src/languages/pf.js) has the `pf.conf` alias.
* [handlebars](https://github.com/isagalaev/highlight.js/blob/905119aad47d4bb3d4fbaa14df7598034dccb6a3/src/languages/handlebars.js) has the `html.hbs` and `html.handlebars` aliases.
* [Objective-C] sets a precedent for using dashes in names and not for scoping (`obj-c` is one of the aliases).

However, there is also `closure` / `closure-repl`, `erlang` / `erlang-repl` and `vbscript` / `vbscript-html`, which I missed before. Perhaps we should go with `julia-repl` then?

There's also one side note regarding Atom -- while the grammar is called `julia-console`, you can't really use it for code blocks in Markdown, or at least I don't know how to select it (neither `julia-console` nor `julia.console` seems to work, although it's possible to use `julia.console` in Markdown preview if the JuliaEditorSupport#106 is applied). TL;DR -- I don't think being consistent with Atom is a very high priority and we could probably just change the name of the grammar there.